src/relay.rs

```diff
@@ -215,10 +218,22 @@ impl<T: DuplexTransport + 'static> Network<T> {
         transport: T,
         account: &str,
         token: &str,
+        token_abi: &str,
         relay: &str,
+        relay_abi: &str,
         confirmations: u64,
     ) -> Result<Self, OperationError> {
-        Self::new(NetworkType::Home, transport, account, token, relay, confirmations, 0)
+        Self::new(
```

Comment:
This is a pretty big function signature but after a quick glance I doubt the wisdom of turning these constructor parameters into separate accessors on that basis.

Add  `#[allow(clippy::too_many_arguments)] ` directive in any case.

Reply:
added thanks